backup_and_restore/application_backup_and_restore/installing/about-oadp-1-3-data-mover.adoc

```diff
@@ -1,27 +1,27 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="about-oadp-1-3-data-mover"]
-= About the OADP 1.3 Data Mover
+= About the {oadp-short} Data Mover
```

Comment:
But we are not removing versions from the software
We are only looking to remove OADP 1.2 documentation because it is no longer supported, as a first step to having properly versioned documentation.